cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -232,7 +232,17 @@ func (c *CVCController) updateCVCObj(
 // 4. Create cstorvolumeclaim resource.
 // 5. Update the cstorvolumeclaim with claimRef info and bound with cstorvolume.
 func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis.CStorVolumeClaim, error) {
-	_ = cvc.Annotations[string(apis.ConfigClassKey)]
+
+	policyName := cvc.Annotations[string(apis.VolumePolicyKey)]
```

Comment:
Better to push getting policy details into a func we can reuse later.

Reply:
done